components/OAuth2.0/authorization.c

```diff
@@ -0,0 +1,283 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "authorization.h"
+#include "OauthHttpLocalServer.h"
+#include "SpiffsManger.h"
+#include "OauthAPICall.h"
+#include "cJSON.h"
+
+// ****************************** Global Variables //FIXME merge structs
+OAuthInterfaceHandler_t *pInterfaceHandler;
+OAuthInterfaceHandler_t OAuthInterfaceHandler;
```

Comment:
you need private hander, please use this method "SpotifyInterfaceHandler_t *InterfaceHandler;
SpotifyPrivateHandler_t PrivateHandler;"

Reply:
We talked about it and Private-handler is not needed.